src/lcm/input_processing/process_model.py

```diff
@@ -94,12 +94,13 @@ def _get_internal_functions(
     # the dynamically generated weighting functions for stochastic next functions, since
     # they are constructed to accept the 'params' argument by default.
 
-    functions = {}
+    functions: dict[str, InternalUserFunction] = {}
+
     for name, func in raw_functions.items():
```

Comment:
```suggestion
    for func_name, func in raw_functions.items():
```
Makes it a bit clearer.